nservicebus/sqlserver/configuration.md

```diff
@@ -46,6 +43,68 @@ By passing the transport a custom factory method which will provide connection s
 
 snippet:sqlserver-custom-connection-factory
 
+## Multiple connection strings
+
+In [*multi-catalog* and *multi-instance* modes](/nservicebus/sqlserver/deployment-options.md) it is necessary to include additional information in configuration.  The sender needs to know the connection string of the receiver, and the subscriber needs the connection string of the publisher.
```

Comment:
It's not very clear.. the current statements say do this, provide this, but doesn't tell you why its needed. Somewhere down below, i had the following suggestion. You can decide if that makes more sense or not. 

If the endpoints are configured to use a different schema, then additional configuration is required for proper message routing. The sending endpoint needs to know the schema information of the receiving endpoint. The same applies when the publishing endpoint and subscribing endpoint use different schemas. The subscriber will need the schema information of the publisher to send subscription requests and to receive events from the publisher.